src/components/HeroesCircles.js

```diff
@@ -0,0 +1,151 @@
+import React from 'react';
+import HerOne from './Images/Component 5.svg';
+import HeroTwo from './Images/Component 1 (1).svg';
+import HeroThree from './Images/Component 6.svg';
+import HeroFour from './Images/Component 4.svg';
+import HeroFive from './Images/Component 3.svg';
+import HeroSix from './Images/Component 2.svg';
+
+function HeroCircles() {
+  const [angle, setAngle] = React.useState(0);
+  const animRef = React.useRef();
+  const cont = React.useRef(null);
+
+  const animate = (time) => {
+    animRef.current = requestAnimationFrame(animate);
+    setAngle((prev) => (prev <= 2 * Math.PI ? prev + 0.005 : 0.005));
+  };
+
+  React.useEffect(() => {
+    animRef.current = requestAnimationFrame(animate);
+    return () => cancelAnimationFrame(animRef.current);
+
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
I would recommend not turning off these messages. Instead, you should do the suggestion. This is the suggestion:

- pass `animate` to the dependency list
- Now you will need to make sure `animate` callback doesn't change each render, how do we make sure of that? there is a hook for it. Pease fix.